Assets/Scripts/Player/Weapons/Weapon.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace OperationBlackwell.Player {
+	public class Weapon: MonoBehaviour {
+		[SerializeField] protected float damage_;
```

Comment:
I don’t think this is part of the coding style currently. It looks good to me, but please make a PR against the documentation stating that this is how we name protected members.